contrib/segyconverter/README.md

```diff
@@ -0,0 +1,47 @@
+# SEG-Y Data Utilities
+
+SEG-Y files can have a lot of variability which makes it difficult to inferer the geometry when converting to npy. The segyio module attempts to do so but fails if there are missing traces in the file (which happens regularly). This utility reads traces using segyio with the inferencing turned off to avoid data loading errors and it uses its own logic to place traces into a numpy array. If traces are missing, the values of the npy array in that location are set to zero
+
+## convert_segy.py script
+
+THe `convert_segy.py` script can work with SEG-Y files and output data on  local disk. This script will process segy files regardless of their structure and output npy files for use in training/scoring. In addition to the npy files, it will write a json file that includes the standard deviation and mean of the original data. The script can additionally use that to normalize and clip that data if indicated in the command line parameters
+
+The resulting npy files will use the following naming convention:
+
+```<prefix>_<inline id>_<xline id>_<depth>.npy```
```

Comment:
Does the user need to conda-activate our repo env to run this code? I think so - please add this step

Reply:
We only need segyio for interaction with SEG-Y and some of common python packages, like:

- os
- math
- pandas
- numpy
- json
- timeit
- argparse

Maybe should we create a requirements file?

Reply:
Hi @GeisaFaustino .. I think most of the libraries you mentioned are included in the Python standard library, so they should come built in. Numpy is included in the cv_lib requirements file: https://github.com/microsoft/seismic-deeplearning/blob/staging/cv_lib/requirements.txt that's installed inside the conda environment (since pip is part of that env as well.. see the environment.yml file:  https://github.com/microsoft/seismic-deeplearning/blob/staging/environment/anaconda/local/environment.yml)

Reply:
+1

Reply:
Added instructions to activate the conda env and referenced back to origian readme for set up